src/pages/DeleteTarget/DeleteTarget.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect, useCallback } from 'react';
+import { useForm } from 'react-hook-form';
+import { useStore } from 'context/Store';
+import useTranslation from 'hooks/useTranslation';
+import { useDeleteTargetMutation } from 'services/target/target';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import Input from 'components/form/Input/Input';
+import Button from 'components/common/Button/Button';
+import Modal from 'components/common/Modal/Modal';
+
+const DeleteTarget = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const t = useTranslation();
+
+  const [state] = useStore();
+
+  const [deleteTarget, { isLoading }] = useDeleteTargetMutation();
+
+  const schema = z.object({
+    area: z.number().min(1, { message: t('deleteTarget.errors') }),
+    title: z.string().min(1, { message: t('deleteTarget.errors') }),
+  });
+
+  const handleClick = () => {
```

Comment:
Would be nice if, after deleting, user was redirected to the main page and/or a success toast were thrown

Reply:
Of course